### DISCUSSION TITLE
netbox current config menu returns error

### DISCUSSION THREAD

**Discussion by sag2643** [netbox-community/netbox#13749] [General]
Hi all I have imported the old database to new one (v3.6.0) and then I have some couple of issues  as below. Also I am not able to multiple scripts under customization-->scripts and I am not sure if managing multiple scripts is possible at all. Can anyone suggest 
![image](https://github.com/netbox-community/netbox/assets/144427799/f68110e5-efb2-4d31-9127-b162e48724e7)

**Comment by deku-m** [comment]
Where do you get the error on what page?
Is there maybe somewhere a custom field created named _meta?

**Comment by candlerb** [comment]
Search the forum for `'NoneType' object has no attribute '_meta'`

You will find a link to #13622 (which occurs when viewing config versions when no initial config version has been created)

This was fixed in Netbox 3.6.1. If you see this error under different conditions, then please explain what you did to trigger the error.

> Also I am not able to multiple scripts under customization-->scripts and I am not sure if managing multiple scripts is possible at all

Multiple scripts work absolutely fine: you simply upload multiple files. Also, each file can also contain multiple scripts.

If you explain your problem more clearly with step-by-step instructions to reproduce what you're seeing, perhaps with screenshots if necessary, someone may be able to help.

**Reply by lasag-2511** [reply]
Hi I have the same behaviour as well
![image](https://github.com/netbox-community/netbox/assets/38073302/033e1cd2-beda-412d-9e35-def8a9e1a42c)

**Reply by candlerb** [reply]
That's not the same behaviour as the original post at all. It's a different exception, and triggered by different cause. (Well, the OP did also mention a problem with multiple scripts, but has not yet provided any details of that problem)

If you have come across a problem, please start a new discussion, and give a description of what you did that triggers the problem.

I would suggest you first upgrade to 3.6.1, as various problems in 3.6.0 have already been fixed.

**Reply by sag2643** [reply]
@candlerb : Thanks for the suggestions, I did upgrade to 3.6.1 and then I am able to see the current config menu .

Regarding the multiple scripts, When I try to add the second script keeping the current one, the second script starts working (active) but the first one doesnot work anymore. I have the below error when i click the first script
```
<class 'extras.models.scripts.ScriptModule.MultipleObjectsReturned'>

get() returned more than one ScriptModule -- it returned 2!

Python version: 3.10.12
NetBox version: 3.6.1
Plugins: None installed
```

**Reply by candlerb** [reply]
Give the specific sequence of steps you're doing, so that the problem is reproducible, then maybe someone can investigate.

Also helpful if you can generate a complete backtrace.

I can only see two places in the source code where this might be returned:

```
./netbox/extras/scripts.py:    module = ScriptModule.objects.get(file_path=f'{module_name}.py')
./netbox/extras/scripts.py:    module = ScriptModule.objects.get(pk=job.object_id)
```

It looks like you could have uploaded two different scripts with the same filename. You can't do that.

**Reply by sag2643** [reply]
yes you are right, i did change the name of the script then it works. Thanks
